### ISSUE TITLE
Null provider message isn't specific

### ISSUE THREAD

**Issue by reubenfirmin** [authzee/kotlin-guice#10]
I'm testing my module using mocks. I'm not supplying a particular property (somewhere). I get the following error message:

```
com.google.inject.CreationException: Unable to create injector, see the following errors:

1) Binding to null instances is not allowed. Use toProvider(Providers.of(null)) if this is your intended behaviour.
  at com.authzee.kotlinguice4.internal.KotlinBindingBuilder.toInstance(KotlinBindingBuilder.kt)

1 error
```

Please ensure you are naming the null property in this message, otherwise it's extremely hard to track down.

**Comment by johnlcox** [comment]
There are no implementations of `toInstance` in kotlin-guice. All `toInstance` methods that are part of the kotlin-guice API directly delegate to the Guice implementation of `toInstance`, which does not add the null property name to the errors.

Unless I'm misunderstanding the issue (a [Minimal Complete Verifiable Example](https://stackoverflow.com/help/mcve) would help), this seems like something that would need to be changed in Guice

**Comment by reubenfirmin** [comment]
Why does it mention a kotlin-guice class (and only that class) in the stack?

You can reproduce this by passing in mocked resources into a module unit test.

**Comment by johnlcox** [comment]
It looks like Guice uses the method [`#getElementSource()`](https://github.com/google/guice/blob/master/core/src/com/google/inject/spi/Elements.java#L530) to get the calling line for `toInstance` bindings when there are errors.

It might be possible to get a better error message in this case by adding some more exclusions to the error source inclusions like this: https://github.com/authzee/kotlin-guice/blob/master/kotlin-guice/src/main/kotlin/com/authzee/kotlinguice4/KotlinModule.kt#L58

**Comment by johnlcox** [comment]
Fixed by #13
